inference-engine/src/vpu/common/include/vpu/ngraph/operations/static_shape_non_maximum_suppression.hpp

```diff
@@ -5,33 +5,27 @@
 #pragma once
 
 #include <ngraph/node.hpp>
-#include <ngraph/op/op.hpp>
+#include <legacy/ngraph_ops/nms_ie.hpp>
 
 #include <memory>
 #include <vector>
 
 namespace ngraph { namespace vpu { namespace op {
 
-class StaticShapeNonMaxSuppression : public ngraph::op::v4::NonMaxSuppression {
+class StaticShapeNonMaxSuppression : public ngraph::op::NonMaxSuppressionIE3 {
```

Comment:
`NonMaxSuppressionIE3` is ambiguous name - why it is a base class here? I expected inheritance fomr NMS-4 or -5.

Reply:
It's because the NMS-5 is purely dynamic. I need to have all the behavior of NMS-5 because it's a generic version of all other NMSs but if I will inherit from NMS-5 (as it was before in this PR) I will need to write the upper-pound propagation function on my own. We already have `NonMaxSuppressionIE3` class which is the analogue of NMS-5 but it propagates the upper-bounds for its outputs (it's done for the plugins which do not support dynamism), so here I try to reuse its functionality.

Reply:
Ok, thanks for explanation